java/ingest/ingest-runner/src/test/java/sleeper/ingest/IngestRecordsFromIteratorIT.java

```diff
@@ -88,6 +94,49 @@ void shouldWriteMultipleRecords() throws Exception {
                 .quantile(0.8, 3L).quantile(0.9, 3L).verify();
     }
 
+    @Test
+    void shouldWriteMultipleRecordsUsingOneReferencePerLeafFileWritingStrategy() throws Exception {
```

Comment:
I'm not sure there's much value in testing both file writing strategies here, since the fact that we're using IngestRecordsFromIterator doesn't seem like it could affect how the files are written? It might be worth reverting these tests to save execution time and simplify the test suite?

Reply:
Thanks, i've removed these new tests